assets/monitoring-plugin/deployment.yaml

```diff
@@ -44,7 +44,27 @@ spec:
             topologyKey: kubernetes.io/hostname
       automountServiceAccountToken: false
       containers:
-      - image: quay.io/openshift/origin-monitoring-plugin:1.0.0
+      - command:
+        - /bin/sh
+        - -c
+        - |
+          # TODO: Maybe just use IPV4 if empty?
+          if [ -z "$POD_IP" ]; then
+            echo "POD_IP env variable is not set or is empty"
```

Comment:
is it even possible?

Reply:
Theoretically if the CNI plugin is broken, yeah, but we'll have some bigger problems then :D . I guess I'll just skip it